applications/FSIApplication/custom_utilities/ibqn_mvqn_convergence_accelerator.h

```diff
@@ -0,0 +1,455 @@
+#define  KRATOS_IBQN_MVQN_CONVERGENCE_ACCELERATOR
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/ublas_interface.h"
+#include "utilities/math_utils.h"
+#include "utilities/qr_utility.h"
+#include "utilities/parallel_utilities.h"
+
+// Application includes
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Forward declaration of standard MVQN
+/// This is required since the include ward avoids the inclusion of the standard MVQN
+template<class TSparseSpace, class TDenseSpace>
+class MVQNFullJacobianConvergenceAccelerator;
+
+/** @brief Interface Block Newton convergence accelerator
+ * Interface Block Newton equations convergence accelerator
+ * @tparam TSparseSpace Linear algebra sparse space
+ * @tparam TDenseSpace Linear algebra dense space
+ */
+template<class TSparseSpace, class TDenseSpace>
+class IBQNMVQNConvergenceAccelerator: public ConvergenceAccelerator<TSparseSpace, TDenseSpace>
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+    KRATOS_CLASS_POINTER_DEFINITION( IBQNMVQNConvergenceAccelerator );
+
+    typedef ConvergenceAccelerator<TSparseSpace, TDenseSpace>              BaseType;
+    typedef typename BaseType::Pointer                              BaseTypePointer;
+
+    typedef typename BaseType::DenseVectorType                           VectorType;
+    typedef typename BaseType::DenseVectorPointerType             VectorPointerType;
+
+    typedef typename BaseType::DenseMatrixType                           MatrixType;
+    typedef typename BaseType::DenseMatrixPointerType             MatrixPointerType;
+
+    typedef MVQNFullJacobianConvergenceAccelerator<TSparseSpace, TDenseSpace>          MVQNType;
+    typedef typename MVQNType::Pointer                                          MVQNPointerType;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Construct a new IBQNMVQNConvergenceAccelerator object
+     * MVQN convergence accelerator Json settings constructor
+     * @param rConvAcceleratorParameters Json string encapsulating the settings
+     */
+    explicit IBQNMVQNConvergenceAccelerator(
+        const double InitialOmega = 0.825,
+        const double AbsCutOff = 1e-8)
+        : mInitialOmega(InitialOmega)
+        , mpConvergenceAcceleratorLeft(Kratos::make_unique<MVQNType>(mInitialOmega, AbsCutOff, true))
+        , mpConvergenceAcceleratorRight(Kratos::make_unique<MVQNType>(mInitialOmega, AbsCutOff, true))
+    {
+    }
+
+    explicit IBQNMVQNConvergenceAccelerator(Parameters rParameters)
+    {
+        Parameters mvqn_default_parameters(R"({
+            "solver_type"            : "IBQN_MVQN",
+            "w_0"                    : 0.825,
+            "abs_cut_off_tol"        : 1e-8,
+            "interface_block_newton" : true
```

Comment:
why is this here?
This is a property of the accelerator, nothing the user can choose
or did I misunderstand sth?

Reply:
You understood it correctly, it is a property of the convergence accelerator. I added it as "information".

Reply:
then I would take it out, I find it confusing if it doesn't actually change anything

Reply:
Done.